packages/cli/README.md

```diff
@@ -240,6 +240,28 @@ Run plugins, collect results and upload the report to the Code PushUp portal.
 
 Refer to the [Common Command Options](#common-command-options) for the list of available options.
 
+#### `history` command
+
+Usage:
+`code-pushup history`
+
+Description:
+Run plugins, collect results and upload the report to the Code PushUp portal for a specified number of commits.
+
+Refer to the [Common Command Options](#common-command-options) for the list of available options.
+
+| Option                   | Type      | Default  | Description                                                      |
+| ------------------------ | --------- | -------- | ---------------------------------------------------------------- |
+| **`--targetBranch`**     | `string`  | 'main'   | Branch to crawl history.                                         |
+| **`--forceCleanStatus`** | `boolean` | `report` | If we reset the status to a clean git history forcefully or not. |
```

Comment:
I'm guessing the default of a boolean flag is `true` or `false`, not `report` :sweat_smile:

Reply:
set to false in options as default and in the docs.